.gitignore

```diff
@@ -45,4 +45,5 @@ node_modules/
 
 result.info
 deploy/*
-deploy-latest/*
\ No newline at end of file
+deploy-latest/*
+packages/*/types
```

Comment:
@jinoosss Why is this in here?

Reply:
I think this should be removed.
I'll modify it so that the type is generated in the build path.